controls/stig_ocp4.yml

```diff
@@ -81,7 +79,6 @@ controls:
   title: OpenShift must generate audit rules to capture account related actions.
   rules:
   - audit_rules_sysadmin_actions
-  - audit_rules_usergroup_modification
```

Comment:
Is this rule removed on purpose? The rule audit-rules-usergroup-modification will be removed from the rhcos4-stig profile with this PR.
```
 % oc get profile rhcos4-stig -o=jsonpath={.rules} | jq -r | grep audit-rules-usergroup-modification
  "rhcos4-audit-rules-usergroup-modification",
  "rhcos4-audit-rules-usergroup-modification-group",
  "rhcos4-audit-rules-usergroup-modification-gshadow",
  "rhcos4-audit-rules-usergroup-modification-opasswd",
  "rhcos4-audit-rules-usergroup-modification-passwd",
  "rhcos4-audit-rules-usergroup-modification-shadow",
% oc get profile upstream-rhcos4-stig -o=jsonpath={.rules} | jq -r | grep audit-rules-usergroup-modification
  "upstream-rhcos4-audit-rules-usergroup-modification-group",
  "upstream-rhcos4-audit-rules-usergroup-modification-gshadow",
  "upstream-rhcos4-audit-rules-usergroup-modification-opasswd",
  "upstream-rhcos4-audit-rules-usergroup-modification-passwd",
  "upstream-rhcos4-audit-rules-usergroup-modification-shadow",
```

Reply:
Yes, rule [audit_rules_usergroup_modification](https://github.com/ComplianceAsCode/content/blob/master/linux_os/guide/auditing/auditd_configure_rules/audit_rules_usergroup_modification/rule.yml) adds the same audit rules as the individual ones.
- -w /etc/group -p wa -k audit_rules_usergroup_modification
- -w /etc/passwd -p wa -k audit_rules_usergroup_modification
- -w /etc/gshadow -p wa -k audit_rules_usergroup_modification
- -w /etc/shadow -p wa -k audit_rules_usergroup_modification
- -w /etc/security/opasswd -p wa -k audit_rules_usergroup_modification

This was causing these rules to be duplicated after remediation is applied.
The logs below are from a profile before this patch. 
```console
$ oc get rems | grep usergroup
rhcos4-stig-master-audit-rules-usergroup-modification                             Applied
rhcos4-stig-master-audit-rules-usergroup-modification-group                       Applied
rhcos4-stig-master-audit-rules-usergroup-modification-gshadow                     Applied
rhcos4-stig-master-audit-rules-usergroup-modification-opasswd                     Applied
rhcos4-stig-master-audit-rules-usergroup-modification-passwd                      Applied
rhcos4-stig-master-audit-rules-usergroup-modification-shadow                      Applied
```

```
sh-5.1# ls -l /etc/audit/rules.d/
total 32
-rw-r--r--. 1 root root  58 Aug 13 13:38 30-etc_group_usergroup_modification.rules
-rw-r--r--. 1 root root  60 Aug 13 13:38 30-etc_gshadow_usergroup_modification.rules
-rw-r--r--. 1 root root  59 Aug 13 13:38 30-etc_passwd_usergroup_modification.rules
-rw-r--r--. 1 root root  69 Aug 13 13:38 30-etc_security_opasswd_usergroup_modification.rules
-rw-r--r--. 1 root root  59 Aug 13 13:38 30-etc_shadow_usergroup_modification.rules
-rw-r--r--. 1 root root 305 Aug 13 13:38 75-audit_rules_usergroup_modification.rules
-rw-------. 1 root root 244 Aug 13 12:18 audit.rules
-rw-r--r--. 1 root root 572 Aug 13 13:38 mco-audit-quiet-containers.rules
```

```console
sh-5.1# chroot /host    
sh-5.1# cat /etc/audit/rules.d/audit.rules
## This file is automatically generated from /etc/audit/rules.d
-D
-b 8192
-f 1
-w /etc/group -p wa -k audit_rules_usergroup_modification
-w /etc/gshadow -p wa -k audit_rules_usergroup_modification
-w /etc/passwd -p wa -k audit_rules_usergroup_modification
-w /etc/security/opasswd -p wa -k audit_rules_usergroup_modification
-w /etc/shadow -p wa -k audit_rules_usergroup_modification
-w /etc/group -p wa -k audit_rules_usergroup_modification
-w /etc/passwd -p wa -k audit_rules_usergroup_modification
-w /etc/gshadow -p wa -k audit_rules_usergroup_modification
-w /etc/shadow -p wa -k audit_rules_usergroup_modification
-w /etc/security/opasswd -p wa -k audit_rules_usergroup_modification
--backlog_wait_time 60000
-a exclude,always -F msgtype=NETFILTER_CFG
-a exclude,always -F msgtype=ANOM_PROMISCUOUS


```